apps/hbl365/src/Main.purs

```diff
@@ -159,6 +159,7 @@ render product =
         , customNewPurchase: Just NewPurchase.render
```

Comment:
Let's ask also for the phone number in `Vetrina.Purchase.AccountForm`. 
Furthermore, do we need to remove some options from the country code dropdown?

Reply:
We still don't have phone updates in Persona module. Though I can add it in this branch.

Reply:
I don't think there's any issues with having a larger list of countries here since this is used only for making digital only subscriptions.